lib/install/loaders/elm.js

```diff
@@ -2,12 +2,11 @@ const { resolve } = require('path')
 
 const isProduction = process.env.NODE_ENV === 'production'
 const elmSource = resolve(process.cwd())
```

Comment:
@klazuka One question, in the installer step we are using `src` as elm source location whereas here it's `process` root. Is this correct?

Reply:
This is correct. The local variable is poorly named. It's actually the path to the directory which contains the `elm.json` project manifest file.

Reply:
thanks I see, maybe we can make that clear :)